engine/lite/request-response/api/src/main/scala/pl/touk/nussknacker/engine/requestresponse/api/RequestResponseDeploymentData.scala

```diff
@@ -2,7 +2,9 @@ package pl.touk.nussknacker.engine.requestresponse.api
 
 import io.circe.generic.JsonCodec
 import pl.touk.nussknacker.engine.api.ProcessVersion
-import pl.touk.nussknacker.engine.api.deployment.{DeploymentData, GraphProcess}
+import pl.touk.nussknacker.engine.api.deployment.DeploymentData
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.engine.marshall.ProcessMarshaller._
 
 //TODO: now we pass process version to runtime, we don't have to use deploymentTime, can be removed
-@JsonCodec case class RequestResponseDeploymentData(graphProcess: GraphProcess, deploymentTime: Long, processVersion: ProcessVersion, deploymentData: DeploymentData)
+@JsonCodec case class RequestResponseDeploymentData(graphProcess: CanonicalProcess, deploymentTime: Long, processVersion: ProcessVersion, deploymentData: DeploymentData)
```

Comment:
`canonicalProcess` instead of `graphProcess`

Reply:
I left it because it is a part of api between designer and req-resp engine and also req-resp engine in this form will be removed